src/coverage_reporter/parsers/coveragepy_parser.cr

```diff
@@ -21,123 +18,54 @@ module CoverageReporter
     end
 
     def matches?(filename : String) : Bool
-      File.open(filename) do |f|
+      valid_file_exists = File.open(filename) do |f|
         f.read_at(0, 15) do |io|
           io.gets.try(&.downcase) == "sqlite format 3"
         end
       end
+
+      check_for_coverage_executable(valid_file_exists)
```

Comment:
IS there a reason to pass the Boolean instead of e.g.

```suggestion
      valid_file_exists && check_for_coverage_executable
```

Reply:
Oh yes, that's way better, thanks! Fixed in 0339f0321c6afb11d224d97c79b08612c51185d0